### PR TITLE
Add Terraria egg to DBH.

### PR DESCRIPTION
**Pull request by awaitlol**
Terraria egg addition.

### CODE REVIEW

**Comment by djohts** [comment]
😂

File: egg-terraria-vanilla.json [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2022-10-25T18:12:06+02:00",
+    "name": "Terraria Vanilla",
+    "author": "dan@danbot.host",
+    "description": "Dig, fight, explore, build! Nothing is impossible in this action-packed adventure game.",
+    "features": null,
+    "docker_images": {
+        "Debian": "ghcr.io\/parkervcp\/yolks:debian"
```
Comment (dominnya) [inline egg-terraria-vanilla.json:13]: ahh yes parkervcp image

**Comment by dominnya** [comment]
https://github.com/DanBot-Hosting/pterodactyl-eggs/blob/67e5daf8973e5989c7782834b358bf39902f1ae7/README.md?plain=1#L2

Don't get me wrong but it's a repository for custom docker images, not parkervcp copycat.

**Comment by DEV-DIBSTER** [comment]
> https://github.com/DanBot-Hosting/pterodactyl-eggs/blob/67e5daf8973e5989c7782834b358bf39902f1ae7/README.md?plain=1#L2
> 
> Don't get me wrong but it's a repository for custom docker images, not parkervcp copycat.

I definitely I agree with this, I'll ask Dan tomorrow about it but there's 99% chance this will be closed.

**Pull request closed without merging**